sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/src/main/java/com/huawei/registry/inject/source/OriginRegistrySwitchSource.java

```diff
@@ -23,7 +23,7 @@
 import java.util.Map;
 
 /**
- * 关闭原始注册中心开关配置源
+ * Turn off the original registry toggle configuration source
```

Comment:
Turn off the original registry switch configuration source

Reply:
done